README.md

```diff
@@ -485,6 +485,24 @@ to run. Since arguments are specified in the user interface via text area
 inputs, it’s important to check that they conform to the format your Task
 expects, and to sanitize any inputs if necessary.
 
+#### Validating Task Parameters
+
+Task attributes can be validated using Active Model Validations. Attributes
+are validated when a Run is created, and validation errors are recorded on the
+Run.
+
+If an attribute uses an inclusion validator with a supported enumerable type,
+the set of values will be used to populate a dropdown in the user interface. The
+following types are supported:
```

Comment:
"recorded" is a bit misleading, we don't persist the errors. Also I don't think we should mention the Run.
```suggestion
Task attributes can be validated using Active Model Validations. Attributes are
validated before a Task is enqueued.

If an attribute uses an inclusion validator with a supported `in:` option, the
set of values will be used to populate a dropdown in the user interface. The
following types are supported:
```